perl/lib/NeedRestart/UI.pm

```diff
@@ -39,6 +39,23 @@ sub new {
     }, $class;
 }
 
+# my $columns = &_get_columns();
+#
+# Return the number of columns to use for output.
+sub _get_columns {
+    my $default_columns = 80;           # Sane default
+    if (-t *STDOUT) {
+        my ($columns) = GetTerminalSize(\*STDOUT);
```

Comment:
Please pass the filehandle twice due to Debian Bug#824564 (as already done in wprint).